src/main/java/io/prestosql/tpch/TpchEntity.java

```diff
@@ -13,9 +13,15 @@
  */
 package io.prestosql.tpch;
 
-public interface TpchEntity
+public abstract class TpchEntity
```

Comment:
I would use `interface` with default method instead.

Reply:
it would not be able to provide toString